rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/nodesource/common/ConfigurableField.java

```diff
@@ -26,6 +26,9 @@
 package org.ow2.proactive.resourcemanager.nodesource.common;
 
 import java.io.Serializable;
+import java.lang.reflect.Field;
+import java.util.HashMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)